### PR TITLE
Remove My Reports -> Map

### LINKED ISSUE #99: Remove My Reports -> Map
Keep only My Reports -> List

### PR DESCRIPTION
**Pull request by DigitalSeneca**
fixes #99 

| My reports | Opened report |
| ------------- | ------------- |
| image | image |

### CODE REVIEW

File: lib/pages/my_reports_pages/my_reports_page.dart [hunk 1]
```diff
@@ -30,7 +28,8 @@ class MyReportsPage extends StatefulWidget {
 }
 
 class _MyReportsPageState extends State<MyReportsPage> {
-  LatLng? location = LatLng(Utils.defaultLocation.latitude, Utils.defaultLocation.longitude);
+  LatLng? location =
+      LatLng(Utils.defaultLocation.latitude, Utils.defaultLocation.longitude);
 
   //Map
   late ClusteringHelper clusteringHelper;
```
Comment (epou) [inline lib/pages/my_reports_pages/my_reports_page.dart:35]: Shouldn't be this removed too?

File: lib/pages/my_reports_pages/my_reports_page.dart [hunk 2]
```diff
@@ -44,24 +43,13 @@ class _MyReportsPageState extends State<MyReportsPage> {
   //My reports
   List<Report> _myData = [];
 
-  //Data
-  Position? _locationData;
-  double? _zoomData;
-
-  final _pagesController = PageController();
-
   StreamController<List<Report>> dataMarkersStream =
```
Comment (epou) [inline lib/pages/my_reports_pages/my_reports_page.dart:46]: This is no longer needed

File: lib/pages/my_reports_pages/my_reports_page.dart
Comment (epou) [inline lib/pages/my_reports_pages/my_reports_page.dart:1]: _initLocation is no longer needed

File: lib/pages/my_reports_pages/my_reports_page.dart [hunk 3]
```diff
@@ -789,22 +614,12 @@ class _MyReportsPageState extends State<MyReportsPage> {
...
-        var location = myData[0].location_choice == 'current'
-            ? LatLng(myData[0].current_location_lat!,
-                myData[0].current_location_lon!)
-            : LatLng(myData[0].selected_location_lat!,
-                myData[0].selected_location_lon!);
-        await mapController.animateCamera(CameraUpdate.newCameraPosition(
-            CameraPosition(target: location, zoom: 15)));
-      }
 
       clusteringHelper.updateData(listMarkers);
```
Comment (epou) [inline lib/pages/my_reports_pages/my_reports_page.dart:618]: remove

**Comment by epou** [comment]
@DigitalSeneca , could you confirm this pull request is staled and has been replaced by #128? In that case, we should close it.

**Comment by DigitalSeneca** [comment]
> @DigitalSeneca , could you confirm this pull request is staled and has been replaced by #128? In that case, we should close it.

Yes, I confirm it

**Pull request closed without merging**